src/core/lib/channel/channel_stack_builder.cc

```diff
@@ -216,6 +216,9 @@ bool grpc_channel_stack_builder_prepend_filter(
 static void add_after(filter_node* before, const grpc_channel_filter* filter,
                       grpc_post_filter_create_init_func post_init_func,
                       void* user_data) {
+  if (grpc_trace_channel_stack_builder.enabled()) {
+    gpr_log(GPR_DEBUG, "adding filter: %s", filter->name);
```

Comment:
If multiple channel stacks are being constructed simultaneously, would it be better to include some context in the log about which stack this filter corresponds to? (it may have other drawbacks, but an alternative to logging here would be to log the filters after the stack is built, e.g., in `grpc_channel_stack_init`)

Reply:
Agreed; this message should start with the address of the builder so that interspersed builders aren't confusing.